.github/workflows/build_packages.yaml

```diff
@@ -334,17 +334,6 @@ jobs:
         profile: ${{fromJSON(needs.prepare.outputs.profiles)}}
         otp:
```

Comment:
if we are only to support one otp. it does not have to be a matrix.

Reply:
This is written so that can use the contextual features of the github action